Source/storm/storm.cpp

```diff
@@ -215,6 +215,41 @@ void setIniFloat(const char *keyname, const char *valuename, float value)
 	getIni().SetDoubleValue(keyname, valuename, value);
 }
 
+/**
+ * @brief Check item exists in comma-sparated list
+ */
+bool IniFound(const char *list, const char *item)

```

Comment:
`IniFound` is not a good function name IMHO. If the intention is to have something that checks whether or not something is in the ini file, I'd suggest `IsInIni` or `ExistsInIni` or even `ExistsInConfiguration` (abstract the fact we use "ini" and not something else).

EDIT: Looking at the implementation, I see _zero_  mentions of INI or even configuration for that matter. This seems to be a general-purpose "search on a list"-type of method. In that case, the function name is even worse here, introducing unnecessary coupling to the config system where there is none.